docs/reference/nodes/variable.md

```diff
@@ -0,0 +1,22 @@
+# Variable Node
+
+The *Variable* node provides a way to save and reuse data throughout the workflow. Plugs set in a *Variable* node will be treated as global variables of the workflow, allowing their values to be accessed at any point in the graph. This node is helpful when multiple nodes across large workflows need to share data.
+
+![Variable Node](../../images/nodes/variable.png)  
+
+### Inputs
+
+When an input plug is added in a *Variable* node, Shift will create an internal workflow global variable with the name of the plug. When the *Variable* node gets executed, the global variable will be updated with the value set in the input plug of the node. 
+
+### Outputs
+
+Output plugs in the *Variable* node are used to retrieve the data from the global variables with a **matching plug name**. When the *Variable* node gets executed, the global variable value will be set as the value of the output plug.

```

Comment:
```suggestion
Output plugs in the *Variable* node are used to retrieve the data from the global variables with a **matching plug name**. When the *Variable* node gets executed, the global variable value will be set as the value of the output plug with matching name.

```

Reply:
We already mention that it must have a matching name in the previous sentence

Reply:
not very clear within the previous section ayways what we talk about before is setting the workflow vaiable with a name matching the input plug name. Here we say that the output plug is set to the value of the matching workflow variable. It is slightly different.

We can skip adding this specification if you do not like it. In my opinion it does no harm to repeat important concepts.

Feel free to resolve with the solution you prefer.